molecule/default/tests/test_create_snapshot_on_instance.py

```diff
@@ -65,10 +62,11 @@ def test_create_instance_from_snapshot(host):
     }
 
     # Boot new instance using the newly created snapshot:
-    utils.create_instance(data_snapshot, host)
+    helpers.create_instance(data_snapshot, host)
+    sleep(30)  # TODO: will find out a better way to avoid implicit sleep.
```

Comment:
Is this sleep call necessary given the retry nature of the `get_expected_value` call below?